tests/transactions/tests.py

```diff
@@ -220,14 +221,12 @@ def test_prevent_rollback(self):
 
     @skipIf(sys.platform.startswith('win'), "Windows doesn't have signals.")
     def test_rollback_on_keyboardinterrupt(self):
-        try:
+        with suppress(KeyboardInterrupt):
```

Comment:
Can this be combined with the line below?

```python
with suppress(KeyboardInterrupt), transaction.atomic():
```

While I've highlighted this one, there are a lot more places where this could be done. It would remove excess levels of indentation, but may be less clear in certain cases, e.g. nested `transaction.atomic()` for savepoints.

I'm not massively pushing for this, just thought I'd put it out there.